pkg/lib/file_test.go

```diff
@@ -0,0 +1,47 @@
+package lib_test
+
+import (
+	"io/fs"
+	"os"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+	"stackerbuild.io/stacker/pkg/lib"
+)
+
+func TestFile(t *testing.T) {
+	Convey("FileCopy", t, func() {
+		src, err := os.CreateTemp("", "src")
+		So(err, ShouldBeNil)
+		So(src, ShouldNotBeNil)
+		defer os.Remove(src.Name())
+
+		_, err = src.Write([]byte("hello world!"))
+		So(err, ShouldBeNil)
+
+		Convey("With defaults", func() {
+			dest, err := os.CreateTemp("", "dest")
+			So(err, ShouldBeNil)
+			defer os.Remove(dest.Name())
+
+			err = lib.FileCopy(dest.Name(), src.Name(), nil, -1, -1)
+			So(err, ShouldBeNil)
+		})
+
+		Convey("With non-default mode", func() {
+			dest, err := os.CreateTemp("", "dest")
+			So(err, ShouldBeNil)
+			defer os.Remove(dest.Name())
+
+			mode := fs.FileMode(0644)
+			err = lib.FileCopy(dest.Name(), src.Name(), &mode, -1, -1)
+			So(err, ShouldBeNil)
+		})
+	})
+
+	Convey("FindFiles", t, func() {
+		files, err := lib.FindFiles("/tmp", ".*")
```

Comment:
Can you create directory with os.MkdirTemp and then put a file in it and then use lib.FindFiles?  that way it won't depend on the assumption that /tmp exists on the system.

Reply:
fixed.